pcp/screens/cgroups

```diff
@@ -0,0 +1,28 @@
+#
+# pcp-htop(1) configuration file - see pcp-htop(5)
+#
+
+[cgroup]
+caption = Cgroup
+enabled = true
+
+cuser.heading = UTIME
+cuser.caption = User CPU Time
+cuser.width = -8
+cuser.metric = 1000 * rate(cgroup.cpu.stat.user)
+
+csystem.heading = STIME
+csystem.caption = Kernel CPU Time
+csystem.width = -8
+csystem.metric = 1000 * rate(cgroup.cpu.stat.system)
+
+cusage.heading = CPU
+cusage.caption = CPU Time
+cusage.width = -8
+cusage.metric = 1000 * rate(cgroup.cpu.stat.usage)
```

Comment:
@smalinux let's make this a percentage utilization - currently it's showing the sum of the other two (user / kernel) columns (i.e. its essentially repeating existing info there).
```
-cusage.heading = CPU
-cusage.caption = CPU Time
-cusage.width = -8
-cusage.metric = 1000 * rate(cgroup.cpu.stat.usage)
+cusage.heading = CPU%
+cusage.caption = CPU Utilization
+cusage.width = 6
+cusage.metric = 100 * (rate(cgroup.cpu.stat.usage) / hinv.ncpu)
```